src/notation/view/widgets/editstaff.ui

```diff
@@ -391,7 +420,7 @@
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
-             <enum>Qt::Horizontal</enum>
+             <enum>Qt::Orientation::Horizontal</enum>
```

Comment:
Changes like these are automatically made by recent versions of Qt Creator. They seemed to cause layout issues in other dialogs, so there I had to undo them manually. Perhaps that's not the case here, but it would be good to double-check that there is no weird sizing behaviour when resizing the dialog, also in comparison with the current master branch.

Reply:
I can't see any difference in the resizing behavior between the released 4.4 and the dev branch with these changes. I'll back them out if you wish, but there are several of them.

Reply:
This also could be a platform dependency. I'll try it on Win.

Reply:
Nah, I think it's fine. I also tried it quickly myself, and can confirm that there is no strange behaviour.